src/job/job.py

```diff
@@ -695,7 +695,6 @@ def run_job_docker_compose(self, c):
             c.execute(['docker-compose', '-f', compose_file_new, 'up',
                        '--abort-on-container-exit'], env=self.environment, show=True, cwd=cwd)
             c.execute(['docker-compose', '-f', compose_file_new, 'ps'], env=self.environment, cwd=cwd, show=True)
-            c.execute(['get_compose_exit_code.sh', compose_file_new], env=self.environment, cwd=cwd, show=True)
```

Comment:
If this is sufficient to get the expected behavior, the script `get_compose_exit_code.sh` should also be removed including the reference to that file `Dockerfile`.
Please double check I didn't miss other references to that script somewhere.